perun-wui-core/src/main/java/cz/metacentrum/perun/wui/widgets/PerunDataGrid.java

```diff
@@ -77,6 +74,8 @@ public boolean canSelectObject(T object) {
 	Map<Column<T, ?>, Header<?>> columnHeaders = new HashMap<>();
 	Map<Column<T, ?>, Header<?>> columnFooters = new HashMap<>();
 
+	Set<PerunColumnType> columnTypeSet = new HashSet<>();
```

Comment:
Hi, please rename variable to be more descriptive of its purpose. Now it describes type of variable, which is clear from its definition. Something like "columnsToFilterBy" should be more understandable, when reading further code.